README.md

```diff
@@ -34,12 +34,18 @@ $ bundle exec rails g solidus_related_products:install
 
 You can create four different RelationTypes:
 
-| Applies From | Applies To |
-| - | - |
-| Spree::Product | Spree::Product |
-| Spree::Product | Spree::Variant |
-| Spree::Variant | Spree::Product |
-| Spree::Variant | Spree::Variant |
+| Applies From | Applies To | Bi-Directional available |
+| - | - | - |
+| Spree::Product | Spree::Product | Yes |
+| Spree::Product | Spree::Variant | No  |
+| Spree::Variant | Spree::Product | No  |
+| Spree::Variant | Spree::Variant | Yes |
+
+**Bi-Directional**
+You can optionally set the bi-directional flag (if available) to automatically create the inverse relation.
+Keep in mind that if you remove one side of the relation, also the other hand will be removed, this is not applied to
+the discount and description of the relation, they both continue to work independently and the update on one side
```

Comment:
How do discounts work for bi-directional relation? What happens if I specify a discount of 50% in the relation from product A to product B, but a discount of 30% in the relation from product B to product A? Also, even if the discounts are the same, are you sure the discount won't be applied twice?

Reply:
https://github.com/solidusio-contrib/solidus_related_products/pull/75#discussion_r404353391